Include/object.h

```diff
@@ -105,6 +105,7 @@ whose size is determined when the object is allocated.
  */
 typedef struct _object {
     PyObject_HEAD
+    Py_ssize_t ob_bstate;
```

Comment:
Don't we want to put this only in strings/bytes, rather than in *every* object? Or is that not possible?

Reply:
Some methods use the abstract object type see: https://github.com/softdevteam/pygrate2/pull/39 , so it is necessary.

Reply:
I'm unsure what it means to ask for the bstate of something that isn't a string. Maybe I'm missing something!